enginetest/queries/queries.go

```diff
@@ -3159,6 +3159,14 @@ var QueryTests = []QueryTest{
 		Query:    "select date('2022-11-19 11:53:45') & date('2022-11-11 11:53:45');",
 		Expected: []sql.Row{{uint64(20221111)}},
 	},
+	{
+		Query:    "select date_add(time('12:13:14'), interval 1 minute);",
+		Expected: []sql.Row{{sql.Timespan(44054000000)}},
+	},
+	{
+		Query:    "select date_sub(time('12:13:14'), interval 1 minute);",
+		Expected: []sql.Row{{sql.Timespan(43934000000)}},
+	},
```

Comment:
(minor) Might be nice to place these queries next to the other `date_add`/`date_sub` queries around line 5143, just to try and keep this query list a little more organized.

Reply:
I think we have a way to automatically generate the gms README, but not update the sql reference on our docs.
I'll make a PR marking `TIME()` as supported now on the docs repo.